### PR TITLE
Merge development changes from this week into the main branch.

### PR DESCRIPTION
**Pull request by mnichols08**
Please review the changes so we can merge the work we completed this week into the main branch =)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -11,51 +61,64 @@ These instructions will get you a copy of the project up and running on your loc
...
+- **Chingu Voyages** - _Initial work_ - [chingu-voyages](https://github.com/chingu-voyages)
+- **Emilie Echevin** -- _Developer_ - [Emimint](https://github.com/Emimint)
+- **Emilio Rivera** -- _Developer_ - [emilio12345](https://github.com/emilio12345)
+- **Gabriela de Paula** -- _Developer_ - [PaulaR-05](https://github.com/PaulaR-05)
+- **Jim Medlock** - _Initial work_ - [jmedlock](https://github.com/jdmedlock)
+- **Mikey Nichols** -- _Developer_ - [mnichols08](https://github.com/mnichols08)
+- **Nazile Tag** -- _Developer_ - [Nazile-Tag](https://github.com/Nazile-Tag)
 
-See also the list of [contributors](/CONTRIBUTORS.md) who participated in this project. 
+See also the list of [contributors](/CONTRIBUTORS.md) who participated in this project.
```
Comment (mnichols08) [inline docs/README.md:114]: I think we may have to change `/CONTRIBUTORS.md` to `/docs/CONTRIBUTORS.md` to fix the link.

I think we should still merge this code but create an bug report to fix this in another PR

File: src/utils/data/funnies.json
Comment (mnichols08) [inline src/utils/data/funnies.json:1]: This is an easy feature to contribute to. These are the random phrases that show up and we can extend it by adding more strings within this array.

**Review by mnichols08 — COMMENTED** [review]
I've left individual notes in a few places, please review so we can push into the main branch

**Review by Emimint — APPROVED** [review]
All good 👍 ~!

**Review by PaulaR-05 — APPROVED** [review]

**Pull request merged**